apps/the-au-pair/src/app/app.module.ts

```diff
@@ -1,17 +1,12 @@
 import { NgModule } from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
-import { RouteReuseStrategy } from '@angular/router';
-
-import { IonicModule, IonicRouteStrategy } from '@ionic/angular';
-
+import { IonicModule } from '@ionic/angular';
+import { ShellModule } from '@the-au-pair/shell/feature';
 import { AppComponent } from './app.component';
-import { AppRoutingModule } from './app-routing.module';
 
 @NgModule({
   declarations: [AppComponent],
   entryComponents: [],
-  imports: [BrowserModule, IonicModule.forRoot(), AppRoutingModule],
-  providers: [{ provide: RouteReuseStrategy, useClass: IonicRouteStrategy }],
+  imports: [IonicModule.forRoot(), ShellModule],
   bootstrap: [AppComponent],
 })
-export class AppModule {}
+export class AppModule {}
```

Comment:
Please add a new line, we have not implemented linting tests yet

Reply:
Updated